lib/dependencies.sh

```diff
@@ -1,3 +1,7 @@
+measure_size() {
+  echo "$((du -s node_modules 2>/dev/null || echo 0) | awk '{print $1;}')"
```

Comment:
don't need the `;` unless you are going to add another command. May want to add `exit` to ensure you are processing only the first line, although that's all you should be getting anyway.